src/content/app/genome-browser/hooks/useGenomeBrowser.ts

```diff
@@ -151,6 +152,78 @@ const useGenomeBrowser = () => {
     }
   };
 
+  const restoreTrackConfigStates = () => {
+    if (!activeGenomeId || !genomeBrowser) {
```

Comment:
No need to use the question mark after `genomeBrowser` in the code below then.

Reply:
✅